scripts/setup.sh

```diff
@@ -212,54 +208,20 @@ SERVING_APP_ENGINE=$(echo "$APP_ENGINE" | "$JQ" -r '.servingStatus')
 
 if [[ -z "$APP_ENGINE" ]]; then
   echo
-  echo "To continue deployment your GCP project must contain an App Engine app - it's required for Cloud Scheduler"
-  while true; do
-    read -p "Do you want to create App Engine app? [y/n]" yn
-    case $yn in
-    [Yy]*)
-      echo
-      echo "Please provide the region for App Engine app."
-      echo
-      echo "Available regions:"
-      APP_ENGINE_LOCATIONS=$(gcloud app regions list --format="json" | "$JQ" -r '.[] | .region')
-      echo "$APP_ENGINE_LOCATIONS"
-      readarray -t LOCATIONS_ARR <<<"$(echo "${APP_ENGINE_LOCATIONS}")"
-      echo
-      while ! [[ " ${LOCATIONS_ARR[*]} " == *" $APP_ENGINE_LOCATION "* ]]; do
-        read -p "Enter location for App Engine app: " -e APP_ENGINE_LOCATION
-      done
-      echo
-      echo "- creating the App Engine app"
-      gcloud app create -q --region="$APP_ENGINE_LOCATION"
-      break
-      ;;
-    [Nn]*)
-      echo
-      echo -e "\e[91mERROR: \e[37mCannot continue without App Engine. Deployment aborted."
-      exit
-      ;;
-    *) echo "- please answer y or n" ;;
-    esac
-  done
+  echo "AppEngine not found. It will be created in your preferredAppEngineRegion $PREFERRED_APP_ENGINE_REGION"
```

Comment:
no can't be like this. You can create only one App Engine on whole subscription and only one. If you will to in that way you can break customer subscription

Reply:
But this line above "if [[ -z "$APP_ENGINE" ]]; then", this is a check for this, right? 
If there is app engine no action. If no app engine, then creating app engine. 
Am I missing something?

Reply:
thats true but, APPEngine can be only one on whole subscription an after creation it can't be deleted. Also it could be in the same region but it is not required. better will be an error if not exits then creating by default in some region without client decision

Reply:
+ there is inconsistency in naming for two regions:
`Two locations, which are called europe-west and us-central in App Engine commands and in the Google Cloud Console, are called europe-west1 and us-central1`
https://cloud.google.com/appengine/docs/locations

That's why we list all available app engine regions with proper naming in console for client.

Reply:
I talked to @karolkieglerski and we agreed on following solution:
1. add explicit functionRegion. Right now function implicitly takes region from cloud shell
2. AppEngine region - parameter stays, will add comment that this region is funky and will give link to AppEngine regions
3. The code for Appengine stays as it is now, i.e. if AppEngine not found, create one in preferredAppEngine region

We also talked about gcloud quiet by default (right now gcloud is still interactive; for example in our pipeline we use -d option of our script to make gcloud quiet, but default is to have gcloud interactive), but not sure if I will pursue it here.

@mswiatkowska does it look ok for you?

Reply:
Yes, it looks ok to me. We considered using gcloud quiet by default but in the end we decided to leave interactive mode as default not to override e.g. roles quietly... not sure if it should be changed.

Reply:
The role in 99.9% will be the default "dynatrace_extension.metrics", I think we can override it freely